style.css

```diff
@@ -0,0 +1,352 @@
+body{
+    margin: 0rem;
+    padding: 0rem;
+    font-family: sans-serif;
+}
+:root{
+    font-size: 10px;
+}
+.navbar{
+    height: 8rem;
+    width: 100%;
+    justify-content: space-between;
+    border-bottom: 0.12rem solid #eee;
+    font-family: Arial, Helvetica, sans-serif;
+}
+.display-flex{
+    display: flex;
+    align-items: center;
+}
+.navbar__logo{
+    border-right: 0.12rem solid #eeee;
+    font-size: 4.5rem;
+    letter-spacing: -0.3rem;
+    font-weight: bold;
+    padding-right: 3rem;
+    padding-left: 3rem;
+}
+.navbar__logo-yellow{
+    color: #f8cb46;  
+}
+.navbar__logo-green{
+    color: #54b226;
+}
+.navbar__delivery-details{
+    margin-left: 4.5rem;
+}
+.navbar__delivery-time{
+    font-weight: bold;
+    letter-spacing: 0.05rem;
+    font-size: 1.7rem;
+}
+.navbar__location{
+    font-size: 1.34rem;
+    letter-spacing: 0.12rem;
+}
+.navbar__location:hover{
+    cursor: pointer;
+}
+
+.navbar__login{
+    color: #494545;
+    font-weight: lighter;
+    font-size: 1.67rem;
+    padding: 1.5rem;
+    border: none;
+    background: white;
+}
+.navbar__login:hover{
+    border-radius: 0.5rem;
+    cursor: pointer;
+    background-color: #dadada;
+    transition-duration: 0.5s;
+}
+.navbar__mycart{
+    background-color: rgb(12, 131, 31);
+    padding: 1.5rem;
+    color: white;
+    margin: 0rem 4.48rem 0rem 4.5rem;
+    border-radius: 0.75rem;
+    font-size: 1.51rem;
+    font-weight: bold;
+    border: none;
+}
+.navbar__mycart:hover{
+    cursor: pointer;
+}
+.navbar__mycart i{
+    font-size: 1.68rem;
+    margin-right: 0.9rem;
+}
+.category-list{
+    margin: 0rem;
+    padding: 0rem;
+    justify-content: center;
+    font-size: 1.51rem;
+    font-family: Arial, Helvetica, sans-serif;
+    color: #494545;
+    height: 6rem;
+    border-bottom: 0.15rem solid #d6d6d6;
+    list-style: none;
+    letter-spacing: 0.07rem;
+    color: #6a6868;
+}
+
+.category-list__li{
+    height: 100%;
+    padding: 0rem 1.5rem;
+}
+.category-list__li:hover{
+    cursor: pointer;
+    background-color: #8080801c
+}
+
+.products-container{
+    width: 111.94rem;
+    margin: auto;
+    display: grid;
+    grid-template-columns: auto auto auto auto auto;
+    border: 0.15rem solid #d6d6d6;
+    border-top: 0rem;
+    border-radius: 0rem 0rem 0.45rem 0.45rem;
+}
+.products-container__category{
+    grid-column-start: 1;
+    grid-column-end: 2;
+    margin: 0rem;
+    padding: 0rem;
+}
+.products-container__product-image{
+    height: 4.48rem;
+    width: 4.48rem;
+    border-radius: 0.75rem;
+    border: none;
+    background-color: #f8f8f8;
+    display: flex;
```

Comment:
Why flex is required?

Reply:
In the image div, the image is placed at the bottom center of the outer div. To achieve that I used display:flex. I could have used display_flex class as well which I have used in the rest of the elements, but here we needed align-items:end instead of align-items: center (which is defined in the display_flex class). So to avoid overwriting I used display:flex in this class itself.